Makefile

```diff
@@ -131,3 +121,10 @@ build_java: # Required by CI
 	cd sdk/java && gradle --console=plain build
 
 install_java_sdk: # Required by CI
+
+# Pulumi for codegen
+
+.pulumi/bin/pulumi: PULUMI_VERSION := $(shell cat .pulumi.version)
```

Comment:
I'm always scared that this will never get bumped, but its no worse then we have in other places.

Reply:
Yup, I think we just need this as part of the scheduled upgrade job. Have added an issue to track this: https://github.com/pulumi/ci-mgmt/issues/940

Me and @blampe have been debating this area also in https://github.com/pulumi/pulumi-docker-build/pull/68 and my efforts so far have been to standardise on this very simple style of installing an isolated copy of pulumi, and I hope we can also move all bridged providers to this too once the docs issues are resolved and we no longer have to modify the schema for each language.